CONTRIBUTING.md

```diff
@@ -23,34 +23,62 @@ If you find a bug anywhere, you can fix it.
 
 # Setup
 
-1. Fork this repository.
+1. Fork this repository using the **Fork** option at the top-right corner of this page. This will create your own copy of this repository. You'll be redirected to your forked repository. Copy the link of this repository (which will look like `https://github.com/<your-username>/website-welcome/`) as you'll need it in the step 2.
 
-2. Clone your forked repository.
+![how-to-fork](https://i.imgur.com/VfoTxmy.png)
+
+2. Clone your forked repository, this will download your copy of repository in your computer. To do this, open your terminal (command prompt/bash/git bash) and enter the following command, paste your link after the word **clone** without the **<>**.
 ``` 
-git clone https://github.com/<your-username>/website-welcome/ 
+git clone <link which you copied in the step 1>
 ```
 
-3. Add the original repository as a remote repository.
+3. Add the original repository as a remote repository, so that you can anytime pull the latest changes from the main repository which is being deployed, this needs to be done only for the first time.
```

Comment:
can we have a separate sentence instead of comma.
_is being deployed, this needs to be done_